Marlin/src/core/serial.h

```diff
@@ -291,7 +291,7 @@ extern uint8_t marlin_debug_flags;
 
 #define SERIAL_ECHO_TERNARY(TF, PRE, ON, OFF, POST) serial_ternary(TF, PSTR(PRE), PSTR(ON), PSTR(OFF), PSTR(POST))
 
-#if SERIAL_FLOAT_PRECISION
+#if SERIAL_FLOAT_PRECISION > 2
```

Comment:
Shouldn't it need to work with 0-2 too?
What do you think?

Reply:
The move I removed tested _MIN(SERIAL_FLOAT_PRECISION, 2).
When missing define it defaults to old 2 digit. Not sure 1 digit has sense

Reply:
Forgot about that, ignore me... 😋

Reply:
The move I removed tested _MIN(SERIAL_FLOAT_PRECISION, 2).
When missing define it defaults to old 2 digit. Not sure 1 digit has sense